### PR TITLE
XW-980 remove global navigation 2015

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
LGTM

File: front/main/app/styles/component/_side-nav.scss [hunk 1]
```diff
@@ -2,121 +2,14 @@
...
-
-		.overlay {
-			opacity: $overlay-opacity;
-			transition: opacity $transition-length;
-			visibility: visible;
-		}
-	}
 }
 
 /// 2016!
```
Comment (rogatty) [inline front/main/app/styles/component/_side-nav.scss]: Do we still need it?

File: front/main/app/templates/components/side-nav-local-navigation-root.hbs [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
-	{{side-nav-local-navigation-menu-2016
-		collapse=this.attrs.collapse
-		loadRandomArticle=this.attrs.loadRandomArticle
-		localNavContent=localNavContent
-		updateContent=(action 'updateContent')
+	{{side-nav-local-navigation-menu
+	collapse=this.attrs.collapse
+	loadRandomArticle=this.attrs.loadRandomArticle
+	localNavContent=localNavContent
+	updateContent=(action 'updateContent')
```
Comment (rogatty) [inline front/main/app/templates/components/side-nav-local-navigation-root.hbs]: Let's keep the indentation pretty.

File: front/main/app/templates/components/side-nav.hbs [hunk 3]
```diff
@@ -1,56 +1,24 @@
...
-			query=searchQuery
-			isInSearchMode=isInSearchMode
-			collapseSideNav='collapse'
-		}}
-		{{local-nav-menu
-			sideNavVisible=shouldBeVisible
-			loadRandomArticle='loadRandomArticle'
-			collapseSideNav='collapse'
-		}}
+<div class='side-nav-menu'>
```
Comment (rogatty) [inline front/main/app/templates/components/side-nav.hbs]: https://github.com/Wikia/guidelines/tree/master/Handlebars#quotes

**Pull request merged**